File: executioner/test/sample_source_code/rust@1.59/test_mle.rs
use std::io;

fn main() -> io::Result<()> {
    let mut buffer = String::new();
    let stdin = io::stdin();
    stdin.read_line(&mut buffer)?;
    let mut bulk = Vec::new();
    while true {
      bulk.push("Bulk");
    }
    Ok(())
}

File: executioner/test/sample-submissions/nodejs-12.22/test_mle.js
const rl = require('readline').createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.on('line', (input) => {
  const bulk = ['bulk'];
  while (true) {
    // Because nodejs times out before running out memory sometimes
    // And unexpected behaviour whilst testing sucks
    bulk.push(bulk.slice(0));
    bulk.push(bulk.slice(0));
    bulk.push(bulk.slice(0));
    bulk.push(bulk.slice(0));
    bulk.push(bulk.slice(0));
    bulk.push(bulk.slice(0));
  }
});
